src/config/gremlins/gremlins.spec.ts

```diff
@@ -1,10 +1,15 @@
 import { START_CAPACITY } from '../../gameConstants';
-import { advanceGameToRound, getGame } from '../../lib/testHelpers';
+import {
+  advanceGameToRound,
+  getGame,
+  testCurrentRound,
+  testFutureRounds,
+} from '../../lib/testHelpers';
 import { gremlins } from './gremlins';
 
 /* Disable round, game and action effects */
 jest.mock('../../state/rounds/getRoundEffects', () => ({
-  getRoundEffects: () => [{ capacityChange: 10 }],
+  getRoundEffects: () => [{ capacityChange: 10, userStoryChange: 30 }],
```

Comment:
Took me a while to find this: We disable all round, game and action effects here in order to test the gremlin effects in isolation (otherwise you'd need to include changes made by foreign effects in you expectations)

Since we had no explicit userStoryChange set, it was indeed `0`

Reply:
Of course. Now I suspect you know me well enough to know that I will immediately use the constants here.

Reply:
...and 20 minutes later I learn that jest won't allow me to use out of scope variables. Me sad.